src/components/layermanager/partials/layermanager.html

```diff
@@ -23,11 +23,12 @@
                 ng-options="(i | gaTimeLabel:layer) for i in layer.timestamps"></select>
       </div>
       <button class="ga-icon ga-btn icon-user"
-              ng-click="showWarning(layer)"></button>
+                ng-click="showWarning(layer)"></button>
       <!--button ng-if="isBodLayer(layer)" class="ga-icon ga-btn icon-info-sign"
-              ng-click="displayLayerMetadata($event, layer)"></button-->
-    </div>
-    <div class="ga-layer-tools">
+                ng-click="displayLayerMetadata($event, layer)"></button-->
+      <button class="ga-icon icon-gear" ng-click="collapsed=!collapsed" href="#subSelection"></button>
+    </div><!--ga-layer-infos-->
+    <div id="subSelection" class="ga-layer-tools" ng-click="collapsed">
```

Comment:
`ìd` attribute must be a unique identifier and here you are in a `ng-repat` directive so this `ìd` will not be unique, so you can remove it.

The `ng-click` is useless. 
Instead of modifying a scope variable with the `ng-click` and rely on it,  I suggest you do all by css.
As you have seen a `.ga-layer-folded` css class is added when the `ga-layer-tools` div element is hidden and this class is removed when it is shown. 

So you just have to replace the appearance effect of the layer tools from right-left to up-down. 
To summarize here are the 2 things to do: 
  - 1st,  hide the `ga-layer-tools` behind the  `ga-layer-infos` element when the `ga-layer-folded` class is set
  - 2nd, replace the right-to-left transition appearance effect of the `ga-layer-tools`element  to an up-to-down transition and increase the size of the `li` in the same time.